libs/net/dal/Services/ContentService.cs

```diff
@@ -174,7 +174,34 @@ public IPaged<Content> FindWithDatabase(ContentFilter filter, bool asNoTracking
         var items = query?.ToArray() ?? Array.Empty<Content>();
         return new Paged<Content>(items, filter.Page, filter.Quantity, total);
     }
+    /// <summary>
+    /// Find most recent front pages (5 of them).
+    /// </summary>
+    /// <returns> Up to 5 front pages.</returns>
+    public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindFrontPages(string index)
+    {
+        var productQueries = new List<Func<QueryContainerDescriptor<API.Areas.Services.Models.Content.ContentModel>, QueryContainer>>();
+        var today = DateTime.Today.ToUniversalTime();
+        productQueries.Add(q => q.Raw(@"{""match"": {""productId"": 10}}"));
+         var response = await _client.SearchAsync<API.Areas.Services.Models.Content.ContentModel>(s =>
```

Comment:
Not a big deal, but you chose to use the NEST library instead of sending a raw JSON request.